tests/test_bot.py

```diff
@@ -750,7 +753,7 @@ async def make_assertion(url, request_data: RequestData, *args, **kwargs):
             else:
                 button_dict = {
                     "text": "button_text",
-                    "web_app": {"url": "https://python-telegram-bot.org"},
+                    "web_app": {"url": "https://python-telegram-offline_bot.org"},
```

Comment:
```suggestion
                    "web_app": {"url": "https://python-telegram-bot.org"},
```
offline ptb would be interesting